Makefile

```diff
@@ -45,6 +45,10 @@ build:
 	@echo "  >  \033[32mBuilding binary...\033[0m "
 	GOBIN=$(PWD)/build/bin go run scripts/ci.go install
 
+# init: Init the gossamer folder using  gssmr0.json genesis file and default config file
+init:
+	./build/bin/gossamer init --genesis config/gssmr0.json --verbosity debug --config ./config.toml
+
```

Comment:
`--genesis config/gssmr0.json` is the default, is it needed here?

Reply:
yes, the obvious should be mentioned to allow for change in case you want to do it differently. That is the whole point of adding the task to the Makefile.